### ISSUE TITLE
ember-data link on implementations page is broken

### ISSUE THREAD

**Issue by conormag** [json-api/json-api#1706]
didnt check any others - just ember-data. current link is https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fjson-api

**Comment by jelhan** [comment]
What do you mean by broken? Implementation page contains the following link for Ember Data: https://github.com/emberjs/data That link points to the GitHub repository used by that project. We could include the API documentation for one of its classes as well. But for most other implementations links to the GitHub repositories are used as well. I tend towards keeping that for consistency.

**Comment by conormag** [comment]
yes sorry it was unclear. there is a link in the second sentence to the "official adapter" which points to the ember-data adapter in the ember docs. the ember api docs give a 404 for that link. it is now https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fadapter%2Fjson-api
